docs/development/build-instructions-linux.md

```diff
@@ -73,7 +73,14 @@ Downloading certain files can take a long time. Notice that we are using
 
 ```sh
 $ cd electron
-$ ./script/bootstrap.py --verbose
+$ npm run bootstrap
```

Comment:
I think we should be clearer on what leaving out the `:dev` sub does.

Suggestions welcomed on this, but as a starter:

##

To bootstrap for a static, non-developer build, run:

```sh
$ cd electron
$ npm run bootstrap
```

Or to bootstrap for a development session that builds faster by not statically linking:

```sh
$ cd electron
$ npm run boostrap:dev
```

Reply:
you're definitely correct, this is far better!